docs/en/operation/Performance-Tuning.md

```diff
@@ -246,6 +259,17 @@ Alluxio workers matches the the number of HDFS datanodes, set
 `alluxio.worker.ufs.instream.cache.max.size=<value of HDFS setting dfs.datanode.handler.count>`
 under the assumption that the workload is spread evenly over Alluxio workers.
 
+## Job Service Tuning
+
+### Job Service Capacity
+Job service limits the total number of currently running jobs to control its resource usage. 
+If jobs tend to be created in large batches, consider increasing `alluxio.job.master.job.capacity` to a larger value than the default 100K. 
+
+### Job Service Throughput
+When there are many concurrent jobs running, and a higher throughput is desired, consider increasing `alluxio.job.worker.threadpool.size` configuration. 
+This allows each job worker to run with more parallel threads. The drawback is it will compete for resources on the worker machines. 
+Recommend 2x core count if most jobs are running in off peak hours only and 1/2 to 1x core count if jobs are running in all hours. 
```

Comment:
maybe link to the CLI option to set this for a specific job as an alternative

Reply:
we don't have that.